src/wmtk/operations/tri_mesh/VertexTangentialLaplacianSmooth.cpp

```diff
@@ -5,6 +5,14 @@
 #include <wmtk/utils/mesh_utils.hpp>
 #include "VertexLaplacianSmooth.hpp"
 
+namespace wmtk::operations {
+void OperationSettings<tri_mesh::VertexTangentialLaplacianSmooth>::initialize_invariants(
```

Comment:
That is quite a bad sign if this function is never called. I think it should be called for every operation at least once. We really need to refactor this initialization stuff.. But that's not in the scope of this PR.